jaxrs/src/main/java/org/killbill/billing/jaxrs/json/SubscriptionJson.java

```diff
@@ -141,7 +141,7 @@ public EventSubscriptionJson(final SubscriptionEvent subscriptionEvent, @Nullabl
             this.eventId = subscriptionEvent.getId();
             this.billingPeriod = billingPeriod;
             this.effectiveDate = subscriptionEvent.getEffectiveDate();
-            this.catalogEffectiveDate = plan != null ? plan.getCatalog().getEffectiveDate() : null;
+            this.catalogEffectiveDate = plan != null ? new DateTime(plan.getCatalog().getEffectiveDate()) : null;
```

Comment:
@sbrossie, does this conversion from`Date` to `DateTime` look correct or should the `referenceTime`/`TimeAwareContext` be used?